lightningd/dual_open_control.c

```diff
@@ -206,6 +209,8 @@ rbf_channel_hook_serialize(struct rbf_channel_payload *payload,
 		     payload->feerate_our_min);
 	json_add_num(stream, "funding_feerate_per_kw",
 		     payload->funding_feerate_per_kw);
+	json_add_amount_sat_only(stream, "channel_max",
+				 payload->channel_max);
```

Comment:
Since this converts to msat, and that fails for UINT64_MAX, this channel_max will actually only appear for non-wumbo channels.  Which is probably OK, but should be documented.

Field should also be called "channel_max_msat".

Reply:
ACK. also updated the `channel_max` field that exist on `openchannel2` hook.